hack/arktos-cni.rc

```diff
@@ -161,7 +161,15 @@ install_arktos_containerd() {
 
 install_flannel() {
     local networkcidr="${1:-10.244.0.0/16}"
-    local apiserver_ip=${2:-127.0.0.1}
+    local apiserver_ip=${API_HOST_IP_EXTERNAL:-127.0.0.1}
```

Comment:
what is the necessity to use env var instead of optional secondary arg here? the positional arg seems able to accommodate env var already.

Reply:
The apiserver_ip on worker node should not point to localhost on worker node. It should point to remote master server so that the following code works.

**line 177: sudo env NODE_NAME=${nodename} ./flanneld-amd64 --ip-masq --kube-subnet-mgr --kube-api-url=http://${apiserver_ip}:8080 --net-config-path=/tmp/net-conf.json >/tmp/flanneld.log 2>&1 &**

Reply:
After talking with HongWei, I understand what he means and will keep function install_flannel() unchanged.